### ISSUE TITLE
500 error on login (can't find database)

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/hamlet#347]
## Major issue

Hamlet (Staging) crashes as soon as you try to login.

Replication:
- Go to https://hamlet-staging.zooniverse.org/
- Attempt to Login by clicking the button
- Result: crash page with 500 error

Analysis:
- Based on the logs, ut seems that Hamlet (staging) is trying to connect to the wrong (i.e. old Azure) database!

### Status

Discussion with @camallen indicates this should be quick fix. 👌 Marking this as a _probably low-urgency_ fix as I'm unsure how heavily Subject Assistent + Hamlet is being used at the moment

**Comment by shaunanoordin** [comment]
This issue was resolved by @camallen a... a week back? Thanks Cam!

Problem: DB didn't exist after migration!
Solution: spin up new DB

Side effects: 
- All previous job statuses have been wiped due to the unexpected DB """migration""" but this is of low import, as we didn't have major users relying on the data on Hamlet. (AFAIK. Nobody has reported anything.)
- The now improperly-named [hamlet-staging](https://hamlet-staging.zooniverse.org/) is now connected to **production,** instead of **staging** as it was previously. This is for the best since we want users to actually use production projects, but it means that [subject-assistant](http://subject-assistant.zooniverse.org/) needs to be rejiggered to also use **production**

**Comment by camallen** [comment]
FWIW - I don't think i modified which API hamlet connected to - i believe this was always set to the main API, https://github.com/zooniverse/hamlet/blob/56ccfb8a39ae09a1a63ef44f012688db46f794e7/hamlet/zooniverse_auth.py#L11

**Comment by shaunanoordin** [comment]
The staging/production dichotomy for hamlet + subject assistant is confusing me like mad, to be honest.

- Originally, I always thought [hamlet](https://hamlet-staging.zooniverse.org/) + [subject assistant](http://subject-assistant.zooniverse.org/) was connected to _production_
- Then, when I was investigating this 500 issue, I noticed that for both hamlet + subject assistant, when I clicked "Login", I was sent to https://panoptes-staging.zooniverse.org/
- At that point I was like, "Oh! Oh, my assumptions were completely wrong, they're actually connected to staging, I need to check with Grant if our test users encountered issues trying to test their production projects"
- Then, 10 minutes ago, when I tried to "fix" subject-assistant to correctly use env=production, I realised the build code literally says `"build": "BABEL_ENV=production NODE_ENV=production npm run clean && webpack --mode=production"` so now I'm like... wait, whaaat?
- So my first assumption WAS correct, these things are SUPPOSED to connect to **production** _all along..._ so how did they switch to staging instead, when I was looking into this 500 issue (plus the [SA proxy server issue](https://github.com/zooniverse/zoo-ml-subject-assistant/pull/67))??

I'm trying not to spend too long untangling this mystery since it's pretty much resolved, but oh boy is it bugging me.
